teps/README.md

```diff
@@ -122,39 +122,56 @@ design and update the missing part in follow-up pull requests which moves the TE
-     offline (e.g. over [slack](https://github.com/tektoncd/community/blob/main/contact.md#slack) or
-     [tekton-dev](https://github.com/tektoncd/community/blob/main/contact.md#mailing-list)).
-2. Once reviewers have been assigned, they should give initial feedback on the PR by
-   [the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api)
-   at the latest.
-
-Reviewers should use [`/approve`](../process.md#prow-commands) to indicate that they approve of the PR being merged. Once all assigned reviewers
-have approved the PR, the final [`/lgtm`](../process.md#prow-commands) can be added in
-[the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api) or sooner
-by anyone with permission to if needed. If a reviewer has not submitted an `/approve`, this is taken to mean that the
-reviewer either a) hasn't done an initial review yet (see SLO above) or b) wants to see changes in the TEP before
-approving. Whenever possible we will try to get an explicit `/approve` from all assigned reviewers before merging, but
-we can always fall back on [our strict approval requirements](#approval-requirements) if needed.
+Whenever possible, a TEP should be approved by all assignees to the PR.
+We may fall back on the strict requirement of approvals from at least two different companies
+in the case of an unresponsive assignee.
+TEP PRs shouldn't be merged if an assignee has strong objections.
 
 TEP PRs that don't affect the proposed design (such as fixing typos, table of contents,
-adding reference links, or marking the TEP as implemented) do not need to meet the
-[approval requirements](#approval-requirements). A single reviewer can feel free to approve
+adding reference links, or marking the TEP as implemented) do not need to meet these
+approval requirements. A single reviewer can feel free to approve
 and LGTM changes like these at any time. PRs marking a TEP as "implementable" should still meet
 the approval requirements, as this label signifies community agreement that the proposal should be
 implemented.
 
+### TEP Review Process and SLOs
+
+Tekton team members regularly review TEP PRs (those with the "tep" label) during
+the [API working group](https://github.com/tektoncd/community/blob/main/working-groups.md#api).
+  - At this meeting, we try to find assignees to review TEP PRs, discuss any TEP PRs that
+  need discussion, and merge any TEP PRs that have met the
+  [approval requirements](#approval-requirements).
+  - Reviewers assigned during the API working group should aim to give feedback by the
+  next API working group meeting.
+  - This meeting can be a great place to find reviewers for your TEP PR or to hold a discussion on parts
+  of the design, but there's no requirement for TEPs to be assigned to reviewers, discussed, or merged
+  during this time.
```

Comment:
Thanks @lbernick for the updates, appreciate your patience 🙏 

Instead of `but there's no requirement for TEPs to be assigned to reviewers, discussed, or merged during this time.`

We can add an additional bullet with a separate paragraph:

```
The TEP author can find required reviewers as soon as the PR is created. Following are the other great ways to find reviewers and publicize your proposed changes:

- Reaching out to the stakeholders or any community maintainers through the PR comments.
- The "tep" channel on
[slack](https://github.com/tektoncd/community/blob/main/contact.md#slack)
- The [tekton-dev mailing list](https://github.com/tektoncd/community/blob/main/contact.md#mailing-list)
```